monero-outreach-docs/translations/es/merchants/merchant_faq_es.md

```diff
@@ -0,0 +1,58 @@
+
+También podrías pagar comisiones adicionales si el número de la tarjeta es ingresado de manera manual, o si tienes que deslizar la tarjeta, o si es una transacción internacional. También podrás tener comisiones mensuales o anuales por parte del procesador de pagos.
+
+### _¿Son las transacciones en Monero menos costosas que las de Bitcoin?_
+
+Igual de complejas a las de las tarjetas de crédito o débito, las comisiones tienden a ser normales. La tasa de intercambio es una comisión de las tarjetas que son reguladas por ley y es actualizada dos veces al año. Las comisiones de Bitcoin no son para nada normales, se incrementan cuando la red está ocupada. Monero ha tenido comisiones muy bajas y estables desde la introducción de las pruebas Bulletproofs en el 2018.
+
+Las comisiones en Bitcoin están en constante cambio. Este gráfico fue sacado de [bitinfocharts.com/comparison...fees-btc-xmr...](https://bitinfocharts.com/comparison/transactionfees-btc-xmr-sma7.html#1y), pero para tener una mejor comprensión de las comisiones en un lapso de tiempo, [monero.how/monero-transaction-fees](https://www.monero.how/monero-transaction-fees) mantiene una lista de las comisiones más altas y bajas, y un promedio de comisión de las 100 últimas transacciones (tx) en cada red.
+
+### _¿Existe alguna manera sencilla de cambiar los XMR de los cliente en dinero fiduciario?_
+
+La mayoría de los procesadores de pagos de terceros te permiten escoger tu moneda de cambio preferida, incluyendo moneda fiduciaria. Nuestra guía de [Procesadores de pago de Monero](https://www.monerooutreach.org/merchants/monero-payment-processor-guide.html) provee una lista detallada sobre eso. Si estás utilizando un procesador de pago con un host propio o si aceptas pagos solo con tu monedero, deberás usar la casa de cambio  que prefieras y su API para hacer el cambio.
+
+### _¿Cómo reporto este ingreso en mis impuestos?_
+
+Lo mejor es que te asesores con tu contador o tu asesor fiscal. Calcula la compra en tu moneda local para que tu contabilidad esté en orden y que todos los impuestos se apliquen correctamente. Haz el cambio de tu moneda a XMR con la tasa de cambio actual. Este tipo de cambio es muy común en los negocios que existen en las fronteras donde se aceptan dos monedas de dos países diferentes. La transacción se registra en tu moneda local y después se hace la conversión. Cuando ya te hayan pagado con XMR y el cliente se haya llevado su producto, ya tienes el hecho imponible de la compra si llegaras a intercambiar los XMR o cambiarlos por moneda fiduciaria. Este segundo hecho imponible no es exclusivo a una transacción en criptomonedas, incluso una transacción con tarjeta de crédito podría crear un hecho imponible dependiendo de cómo gastes tus fondos.
```

Comment:
Instead imponible, I'd use imputado.

Reply:
I left "imponible" because of its exact definition: 'something that can be taxed or tributed'. 
The word "imputable" means: 'something than can be linked to', in this case, yes a tax but it could change the meaning of a product that can be taxed.
Let me know what you think.

Reply:
you 're right